hummingbot/strategy/pure_market_making/pure_market_making_v2.pyx

```diff
@@ -104,7 +104,8 @@ cdef class PureMarketMakingStrategyV2(StrategyBase):
                  logging_options: int = OPTION_LOG_ALL,
                  limit_order_min_expiration: float = 130.0,
                  status_report_interval: float = 900,
-                 asset_price_delegate: AssetPriceDelegate = None):
+                 asset_price_delegate: AssetPriceDelegate = None,
+                 expiration_seconds: float = NaN):
```

Comment:
Is NaN a good default?

Reply:
current default in base class is NaN so it should be ok